src/components/Message/Message.md

```diff
@@ -21,9 +21,9 @@ The Message component provides the following functions to the rendered component
 - **openThread** Handler to open the thread on current message.
 
 ```js
-import { Message, MessageSimple } from '../components';
+import { Message, MessageSimple } from './';
```

Comment:
rather point to the actual file path instead of the `index` file that has these exported so that we can get rid of a lot of the index files to be able to utilize the vscode file search and not have a bunch of random index files show up